upgrade_test.py

```diff
@@ -412,7 +412,11 @@ def test_upgrade_cql_queries(self):
         self.verify_db_data()
         self.verify_stress_thread(stress_queue)
 
-    def fill_and_verify_db_data(self, note, pre_fill=False, rewrite_data=True):
+    # There is a known scylla issue in truncate, the timeout exception will break the subtest,
```

Comment:
Why the timeout exception break the test?
Due to the error or because the truncate didn't work?

Reply:
Due to the error, the subtest is broke. Then the test data won't be refilled again as expected.

- We refill the test data at the end of fill_db_data verify test.

Reply:
Ok, but the truncate is working even with this timeout.
So why not wrapping it with try, except and continue as usual if we caught this timeout exception.